test/bin/misc-test-cases/mdx-test-require-local-package/dune

```diff
@@ -1,9 +1,9 @@
 (rule
- (targets dune.inc.gen)
+ (alias runtest)
  (deps
-  (package mdx))
```

Comment:
Why is this removed? This is required because of the `ocaml-mdx` and `ocaml-mdx-test` binaries trick.